packages/eslint-plugin/tests/rules/unbound-method.test.ts

```diff
@@ -590,5 +590,22 @@ class OtherClass extends BaseClass {
         },
       ],
     },
+    {
+      code: `
+const values = {
+  a() {},
+  b: () => {},
+};
+
+const { a, b } = values;
```

Comment:
A couple more test cases come to mind:

* `const { a: c } = values;`
* `const { b, a } = values;`